osu.Game/Tests/Visual/PlayerTestScene.cs

```diff
@@ -19,20 +20,24 @@ public abstract class PlayerTestScene : RateAdjustedBeatmapTestScene
         /// </summary>
         protected virtual bool HasCustomSteps { get; } = false;
 
-        private readonly Ruleset ruleset;
-
         protected TestPlayer Player;
 
-        protected PlayerTestScene(Ruleset ruleset)
+        protected OsuConfigManager LocalConfig;
+
+        private readonly Ruleset ruleset;
+
+        protected PlayerTestScene()
         {
-            this.ruleset = ruleset;
+            ruleset = CreatePlayerRuleset();
         }
 
-        protected OsuConfigManager LocalConfig;
-
         [BackgroundDependencyLoader]
         private void load()
         {
+            // There are test scenes using current value of the ruleset bindable
+            // on their BDLs (example in TestSceneSliderSnaking's BDL)
+            Ruleset.Value = ruleset.RulesetInfo;
```

Comment:
I'm a little unusure of this being done in every place. Seems like something that could be missed?

Is there a reason this is done here (`PlayerTestScene`, `EditorTestScene`, `SkinnableTestScene`) rather than inside `OsuTestScene`'s BDL itself? I'm fine with the re-exposed abstracts, it's just this part that seems kinda weird.